### DISCUSSION TITLE
catalogue service

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#746] [Q&A]
i'm not able to find the mistake yesterday i got the categories but today i'm not getting it,
eventhough i ran the frontend and mongodb server once again
![Screenshot (25)](https://user-images.githubusercontent.com/129321520/235302155-b50f1c50-368f-4e74-9397-c7472d3b536d.png)
![Uploading Screenshot (24).png…]()

**Comment by Manju9876** [comment]
hello @nandini965 
well, as per my knowledge, you just make sure that the configuration files and the DNS servers are correct and also re-write the code as per the code standards that make the code DRY.

**Reply by nandini965** [reply]
sorry ,,, my mongodb dns record was wrong now i changed it ,, thanq
